src/webgl/p5.Camera.js

```diff
@@ -176,6 +176,20 @@ p5.prototype.ortho = function(...args) {
 };
 
 /**
+ * Sets a perspective matrix as defined by the parameters.
+ *
+ * A frustum is a geometric form: a pyramid with its top
+ * cut off. With the viewer's eye at the imaginary top of
+ * the pyramid, the six planes of the frustum act as clipping
+ * planes when rendering a 3D view. Thus, any form inside the
+ * clipping planes is rendered and visible; anything outside
+ * those planes is not visible.
+ *
+ * Setting the frustum has the effect of changing the perspective
```

Comment:
Can we change this sentence to: "Setting the frustum changes the perspective of the scene being rendered."